specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkManagerAdminRuleCollectionGet.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "networkManagerName": "testNetworkManager",
+    "ruleCollectionName": "testRuleCollection",
+    "configurationName": "myTestSecurityConfig"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/networkManagers/testNetworkManager/securityAdminConfigurations/myTestSecurityConfig/ruleCollections/testRuleCollection",
+        "name": "testRuleCollection",
+        "type": "Microsoft.Network/networkManagers/securityAdminConfigurations/RuleCollections",
+        "systemData": {
+          "createdBy": "b69a9388-9488-4534-b470-7ec6d41beef5",
+          "createdByType": "User",
+          "createdAt": "2021-01-11T18:52:27Z",
+          "lastModifiedBy": "b69a9388-9488-4534-b470-7ec6d41beef5",
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2021-01-11T18:52:27Z"
+        },
+        "properties": {
+          "displayName": "testRuleCollection",
```

Comment:
Do we have to return the associated rules with this Rule collection as well?

Reply:
No, it is anti pattern